src/routes/analytics/add-panel.ts

```diff
@@ -27,6 +29,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
   path: '/panel',
   body: Joi.object().keys({
     title: Joi.string().required().example('My dashboard'),
+    layouts: layoutsSchema.allow(null).empty(null).default(),
```

Comment:
`.allow(null).empty(null)` is already in `layoutsSchema`. Also, if my comment there makes sense, we can also drop `.default()` I think.